src/metabase/api/card.clj

```diff
@@ -165,9 +136,8 @@
 
 (api/defendpoint GET "/"
   "Get all the Cards. Option filter param `f` can be used to change the set of Cards that are returned; default is
-  `all`, but other options include `mine`, `bookmarked`, `database`, `table`, `recent`, `popular`, :using_model
-  and `archived`. See corresponditng implementation functions above for the specific behavior of each filter
-  option. :card_index:"
+  `all`, but other options include `mine`, `bookmarked`, `database`, `table`, `using_model` and `archived`. See
+  corresponding implementation functions above for the specific behavior of each filterp option. :card_index:"
```

Comment:
Removed the options from this API which read from `ViewLog`, since they were unused